defaults/main.yml

```diff
@@ -51,7 +52,7 @@ prometheus_scrape_configs:
 - job_name: "node"
   file_sd_configs:
   - files:
-    - "{{ prometheus_config_dir }}/file_sd/targets.yml"
+    - "{{ prometheus_config_dir }}/file_sd/*.yml"
```

Comment:
This should not be a glob.

Reply:
Ok, but as I said in https://github.com/cloudalchemy/ansible-prometheus/issues/58#issuecomment-357447135, I don't see any other option. Users who want to tune prometheus to their custom setup will overwrite `prometheus_scrape_configs` and this glob won't be used. However for others this will be quite useful.

Reply:
It may be "useful", but it's considered an anti-pattern, and will encourage the wrong usage patterns.

Reply:
Ok, so it seems I need to write some crazy preflight check to notify (fail playbook) user when he provides wrong config.